packages/xod-fs/test/welcomeToXod.spec.js

```diff
@@ -11,7 +11,7 @@ describe('welcome-to-xod', () => {
   const projectPath = 'welcome-to-xod';
 
   it('should load as a valid project', () =>
-    loadProjectWithLibs([], projectPath, workspace)
+    loadProjectWithLibs([workspace], path.resolve(workspace, projectPath))
```

Comment:
A test with multiple workspaces would be awesome(for example, finding one library in the first workspace, and another in the other one, or "preferring" a version of a library which exists in both workspaces from first workspace, etc), but I am not insisting 😁

Reply:
Added one, but into `loadLib.spec.js`